zio-config/src/test/scala/zio/config/SequenceRoundTripTest.scala

```diff
@@ -0,0 +1,53 @@
+package zio.config
+
+import org.scalacheck.{ Gen, Properties }
+import zio.ZIO
+import zio.config.testsupport.TestSupport
+
+object SequenceRoundTripTest extends Properties("sequence round trip tests") with TestSupport {
```

Comment:
It's getting to the point we should consider using ZIO Test for all the tests.

Reply:
For sure. We will have a separate issue/ and follow up PR for this.